### ISSUE TITLE
E2E tests are failing

### ISSUE THREAD

**Issue by runarmod** [TDT4290-Gr6/math-mate#205]
Since PR #203 modified stuff, certain E2E tests that relied on certain aria-labels and classnames are now failing.
